tests/integ/compiler/test_query_generator.py

```diff
@@ -145,6 +146,7 @@ def test_table_create(session, mode):
         column_names=None,
         mode=mode,
         query=df._plan,
+        creation_source=TableCreationSource.OTHERS,
```

Comment:
can we add a test that test with the TableCreationSource is LargeQueryBreakdown, you might have to a check of the result query with actual query text here